ui/urls.py

```diff
@@ -26,8 +27,17 @@
     path('media/<pk>/edit', views.MediaEditView.as_view(), name='media_item_edit'),
     path('media/<pk>', views.MediaView.as_view(), name='media_item'),
     path('channels/<pk>', views.ChannelView.as_view(), name='channel'),
+    path(
+        'create_playlist',
+        login_required(TemplateView.as_view(template_name="ui/create_playlist.html")),
+        name='create_playlist'
```

Comment:
To fit with other views, this should be called "playlist_create". (One could also argue that the upload view should be called "media_item_create" as well while we're at it...)